src/include/platform/ConfigurationManager.h

```diff
@@ -161,6 +161,9 @@ class ConfigurationManager
 
     virtual CHIP_ERROR GetLocationCapability(uint8_t & location);
 
+    // Set storage path for all configs/data
+    virtual CHIP_ERROR SetStorageSpace(const char *path) {return CHIP_NO_ERROR;};
```

Comment:
why is this no-error when this does nothing?

Reply:
Well, either it needs to do that or it needs at least a Mac implementation, since its called from "linux" code in examples/ that is not actually Linux-only but the override in platform/Linux is in fact Linux-only.  Might need more implementations too, if we wanted to do that.

In either case, it seems like this needs followup issues to investigate what's needed, if anything, to support this on non-Linux platforms that use examples/platform/linux.